src/game.cpp

```diff
@@ -11749,12 +11749,52 @@ bool game::plmove(int dx, int dy)
                         add_msg(m_info, _("You can't move %s while standing on it!"), grabbed_vehicle->name.c_str());
                         return false;
                     }
-                    drag_multiplier += (float)(grabbed_vehicle->total_mass() * 1000) /
-                        (float)(u.weight_capacity() * 5);
-                    if (drag_multiplier > 2.0) {
-                        add_msg(m_info, _("The %s is too heavy for you to budge!"), grabbed_vehicle->name.c_str());
+                    //vehicle movement: strength check
```

Comment:
It might be useful to move this into a separate function, allows you to remove much of the indentation space and reduces the size of the already very large `plmove`.